tests/fv3_conf/cpld_datm_cfsr.IN

```diff
@@ -0,0 +1,26 @@
+mkdir INPUT DATM_INPUT RESTART history MOM6_OUTPUT
```

Comment:
The cpld_datm_cfsr and cpd_datm_gefs essentially identical. It means we need to maintain two files instead of one. I think they should be combined by exporting the variable name of 'cfsr' or 'gefs'  and resolution dependent variables in the actual test (datm_ocn_ice_mx100_cfsr etc). All 4 tests could then be made to call cpld_datm_control.IN.

Reply:
In the next PR, both debug and restart tests will be added and cpld_datm_control.IN will be used.

Reply:
The GNU tests and baselines will be added in future PRs.